<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkManagerDemoWidget</class>
 <widget class="QWidget" name="NetworkManagerDemoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetworkManagerDemoWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edUrl">
       <property name="text">
        <string>https://exlmoto.ru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnGo">
       <property name="text">
        <string>Go!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>About Qt...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>edUrl</tabstop>
  <tabstop>bnGo</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
